### ISSUE TITLE
[Bug] Matomo 5.0.2 : /core/Filechecks.php(153): Deprecated - trim(): Passing null to parameter #1 ($string) of type string is deprecated

### ISSUE THREAD

**Issue by heurteph-ei** [piwik/piwik#22101]
### What happened?

https://forum.matomo.org/t/warning-core-filechecks-php-153/55817
Matomo update from Matomo 5.0.2 to Matomo 5.0.3.
After updating to Matomo 5.03 following message is shown in “Alle Webseiten”:
```
WARNING: /core/Filechecks.php(153): Deprecated - trim(): Passing null to parameter #1 ($string) of type string is deprecated - Matomo 5.0.2 (Module: CoreUpdater, Action: newVersionAvailable, In CLI mode: false)
```

### What should happen?

No warning message.

### How can this be reproduced?

Matomo update from Matomo 5.0.2 to Matomo 5.0.3.
Then go to “Alle Webseiten”:

### Matomo version

5.0.2 > 5.0.3

### PHP version

probably 8.x

### Server operating system

unknown

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by MatomoForumNotifications** [comment]
This issue has been mentioned on **Matomo forums**. There might be relevant details there:

https://forum.matomo.org/t/warning-core-filechecks-php-153/55817/2

**Comment by michalkleiner** [comment]
Thanks for the report @heurteph-ei. I've prepared a small PR addressing this issue.

**Comment by michalkleiner** [comment]
@heurteph-ei thanks again for the report, a fix for this will be in 5.1.0.

**Comment by MatomoForumNotifications** [comment]
This issue has been mentioned on **Matomo forums**. There might be relevant details there:

https://forum.matomo.org/t/error-message-after-update/58411/2
